app/src/main/res/layout/lang_item_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You don't need to wrap `TextView` here with `ConstraintLayout`, just `TextView` itself should suffice